backend/utils/resource-utils.js

```diff
@@ -76,23 +76,6 @@ const geocodeAddress = R.memoizeWith(R.identity, async (address) => {
   return parseGeocodingResponse(response.data);
 });
 
-const latlongToAddress = async function (lat, long) {
-  const apiAddress = `${mapquestURI}reverse?key=${mapquestKey}&location=${lat},${long}&includeRoadMetadata=false&includeNearestIntersection=false`;
```

Comment:
just as a sanity check, this is unused code right? we're not deleting something that shouldn't be deleted?

Reply:
That's correct—thanks for bringing it up. Currently we only geocode address to lat/long i.e. one way, so I figured there's no need to keep a function that we don't need since we can always go back in Git history to retrieve if needed later on.